cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/Conversion.java

```diff
@@ -336,8 +337,8 @@ private void newConvertTransformers(Context context) {
             } else if (ends.size() == 3) {
                 c = new NewThreeWindingsTransformerConversion(ends, context);
             } else {
-                String what = String.format("PowerTransformer %s", t);
-                String reason = String.format("Has %d ends. Only 2 or 3 ends are supported", ends.size());
+                Supplier<String> what = () -> String.format("PowerTransformer %s", t);
```

Comment:
In that specific case, a simple concatenation should be more efficient. What do you think?